dashboard/app/controllers/openai_chat_controller.rb

```diff
@@ -22,18 +22,20 @@ def chat_completion
     # If the content is profane, we skip sending to OpenAI and instead hardcode a warning response on the front-end.
     return render(status: :ok, json: {safety_status: filter_result.type, flagged_content: filter_result.content}) if filter_result && filter_result.type == 'profanity'
 
+    # Fetch the level-specific context: instructions and test files if applicable
+    level_id = params[:levelId]
+
+    level_instructions = ""
+    level_instructions = get_level_instructions(level_id) if !!level_id
```

Comment:
Nit: I think the !! are redundant when used in if statements

Reply:
you're totally right! thanks!